packages/cli/index.ts

```diff
@@ -243,24 +242,16 @@ const main = defineCommand({
 
 runMain(main);
 
-async function importPackageJson(p: string): Promise<Record<string, any>> {
-  const { default: obj } = await import(pathToFileURL(p).href, {
-    with: { type: "json" },
-  });
-
-  return obj;
-}
-
 async function writeDeps(p: string, deps: Map<string, string>) {
   const pJsonPath = path.resolve(p, "package.json");
-  const content = await fs.readFile(pJsonPath, "utf-8");
 
-  const pJson = await importPackageJson(pJsonPath);
+  const pJson = await readPackageJSON();
+  
   hijackDeps(deps, pJson.dependencies);
   hijackDeps(deps, pJson.devDependencies);
-  await fs.writeFile(pJsonPath, JSON.stringify(pJson));
+  await writePackageJSON(pJsonPath, pJson);
 
-  return () => fs.writeFile(pJsonPath, content);
+  return () => writePackageJSON(pJsonPath, pJson);
```

Comment:
This is clean up function so when the command finishes running, we revert the changes, that's why we store `content`

Reply:
[`writePackageJSON`](https://github.com/unjs/pkg-types/blob/608aa5c3df8060e78a4632943bda138e30d4e7e5/src/index.ts#L85) is doing same thing